clarify batch management in the Wallet when the new batch of the credentials was received (#452)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

If the Credential Response is not encrypted, the media type of the response MUST be set to `application/json`.

> When the Wallet receives a new batch of Credentials containing a specific Credential Dataset, the Wallet SHOULD delete any Credentials containing the same Credential Dataset it might have received previously.

Reviewer: I wonder if this is slightly over specific - it doesn't give any advice for the case where the issuer doesn't support batch issuance, nor it is necessarily helpful in the case where the wallet wants a batch of 10 and the issuer only supports batches of 5 (the wallet would hence ask for two batches of 5 and definitely wouldn't delete the first batch of 5 when it got the second batch of 5...).

Maybe this is more generic advice?

```suggestion
When the Wallet receives new Credentials containing a specific Credential Dataset, the Wallet SHOULD delete any Credentials containing the same Credential Dataset it received in previous sessions.
```

(I'm not sure "session" is well defined but can't think of another phrase that means "delete credentials you received a while ago but not ones you received 10 seconds ago".)

Reviewer 2: ```suggestion
When the Wallet receives new Credentials containing a specific Credential Dataset, the Wallet SHOULD delete any Credentials containing the same Credential Dataset it received in previous issuance responses if those Credentials are no longer valid.
```
